.github/workflows/deploy_docs.yml

```diff
@@ -1,8 +1,9 @@
-name: Deploy docs
+name: Build and deploy docs
 
 on:
   push:
     branches: [ main ]
+  pull_request:
```

Comment:
we should separate this into a different PR

Reply:
Posted #240.